integration/v7/push/instances_flag_test.go

```diff
@@ -29,7 +29,7 @@ var _ = Describe("push with instances flag", func() {
 				Eventually(session).Should(Exit(0))
 			})
 
-			time.Sleep(5 * time.Second)
+			time.Sleep(10 * time.Second)
```

Comment:
@gururajsh please consider using a centralized sleep time constant to control timings across the whole test suit.